quinn-proto/src/connection/paths.rs

```diff
@@ -87,3 +91,99 @@ impl RttEstimator {
             .map(|srtt| srtt + cmp::max(4 * self.var, TIMER_GRANULARITY))
     }
 }
+
+// Implements Datagram Packetization Layer Path Maximum Transmission Unit Discovery
+//
+// https://www.ietf.org/id/draft-ietf-tsvwg-datagram-plpmtud-21.html
+pub struct MtuDiscovery {
```

Comment:
I'm in favor of the scoping benefits but would prefer to leave the module inline. Splitting a 200-line file makes life harder rather than easier in my opinion (more jumping around between files).

Reply:
Not a strong preference, so that's fine.